client/src/Timer.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState, useEffect } from 'react';
+import './Timer.css';
+
+function Timer() {
+    const totalSeconds = 120;
+    const [isRunning, setIsRunning] = useState(false);
+    const [remainingSeconds, setRemainingSeconds] = useState(totalSeconds);
+    
+    useEffect(() => {
+        handleCountdown()
+    }, [isRunning]);
+    
+    const handleStart = () => {
+        setIsRunning(true);
+    };
+
+    const handleReset = () => {
+        setRemainingSeconds(totalSeconds)
+        handleStart()
+    }
+    
+    const handleCountdown = () => {
+        let countdown;
+    
+        if (isRunning) {
+            countdown = setInterval(() => {
+                setRemainingSeconds((currentSeconds) => {
+                    if (currentSeconds > 0) {
+                        return currentSeconds - 1;
+                    }
+                    setIsRunning(false);
+                    clearInterval(countdown);
+                    return 0;
+                });
+            }, 1000);
+        }
+    
+        return () => clearInterval(countdown);
+    };
+    
+    
+    const displayTime = (seconds) => {
```

Comment:
For a much clearer name, you can change `displayTime` to `displayTimerString`.

Reply:
I resolved this issue and renamed it as suggested